webapp/channels/src/actions/post_actions.ts

```diff
@@ -140,6 +141,36 @@ function storeCommentDraft(rootPostId: string, draft: null) {
     };
 }
 
+export function submitReaction(postId: string, action: string, emojiName: string) {
+    return (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const state = getState() as GlobalState;
+        const getIsReactionAlreadyAddedToPost = makeGetIsReactionAlreadyAddedToPost();
+
+        const isReactionAlreadyAddedToPost = getIsReactionAlreadyAddedToPost(state, postId, emojiName);
```

Comment:
This can be simplified to:

```suggestion
        const isReactionAlreadyAddedToPost  = makeGetIsReactionAlreadyAddedToPost()(state, postId, emojiName);
```

I think that the readibility is a bit better but 0/5. What do you think?